apache/lp.conf

```diff
@@ -9,6 +14,16 @@
     LogLevel warn
     CustomLog /var/log/apache2/access.log combined
     ErrorLog /var/log/apache2/error.log
+    
+    #uncomment to use SSL
```

Comment:
If I read this correctly (which I may not), SSL is required with this config (in which case these should be uncommented).